src/commands/check.js

```diff
@@ -218,6 +220,47 @@ export const handler = catchAsyncError(async opts => {
         isUpdateFinished = true;
         break;
 
+      case 'commit':
+        updatedModules.push(outdatedModule);
+        setModuleVersion(name, to, packageJson);
+        delete config.ignore[name];
+
+        // Showing the list of modules that are going to be updated
+        console.log(
+          `\n${strong('These packages will be updated:')}\n\n` +
```

Comment:
It will always show a single package, right?

Reply:
Yes, should update the text